modules/cluster-logging-logcli-reference.adoc

```diff
@@ -0,0 +1,29 @@
+// Module is included in the following assemblies:
+//cluster-logging-loki.adoc
+:_content-type: REFERENCE
+[id="logging-logcli-about_{context}"]
+
+= Querying Loki
+
+You can query use Loki's command-line interface `logcli` to query logs.
+
+.Example Application Log Query
+[source,terminal]
+----
+logcli --bearer-token="$(oc whoami -t)" --addr https://lokistack-dev-openshift-logging.apps.periklis-logexp.devcluster.openshift.com/api/logs/v1/application labels
```

Comment:
How about hide periklis from the URL.
It is better to add ca-cert or  --tls-skip-verify=false
```
oc extract cm/lokistack-sample-ca-bundle --to=lokistack --confirm
cat lokistack/*.crt >lokistack_ca.crt
logcli -o raw --bearer-token="${bearer_token}" --ca-cert="lokistack_ca.crt" --addr xxxxxx
```